src/modules/services/submodule/buySiriusCoins/views/ViewServicesStableCoinsCheckStatus.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <div>
+    <div class='lg:w-9/12 ml-2 mr-2 lg:ml-auto lg:mr-auto mt-5 bg-white border border-white rounded-md shadow-lg'>
+      <div class='mt-6 px-6 py-10 filter text-center'>
+        <div class="text-md mb-3">Check Buy Status</div>
+        <div>
+          <div class="mt-10 success_box success success-text" v-if="isSuccess">
+            <div class="font-normal">
+              <div class="md:flex my-3 md:my-1">
+                <div class="md:w-56 text-left md:text-right mr-4">BSC Transaction Hash:</div>
+                <div class="font-semibold text-left"><a :href="remoteExplorerLink + remoteTxnHash" target=_new class="hover:underline">{{ remoteTxnHash.substring(0, 7) + '...' + remoteTxnHash.slice(-7) }} <font-awesome-icon icon="external-link-alt" class="ml-1 w-3 h-3 self-center inline-block"></font-awesome-icon></a></div>
+              </div>
+              <div class="md:flex my-3 md:my-1">
+                <div class="md:w-56 text-left md:text-right mr-4">Sirius Transaction Hash</div>
+                <div class="font-semibold text-left"><a :href="explorerLink + siriusTransactionHash" target=_new class="hover:underline">{{ siriusTransactionHash.substring(0, 7) + '...' + siriusTransactionHash.slice(-7) }} <font-awesome-icon icon="external-link-alt" class="ml-1 w-3 h-3 self-center inline-block"></font-awesome-icon></a></div>
+              </div>
+              <div class="md:flex my-3 md:my-1">
+                <div class="md:w-56 text-left md:text-right mr-4">Status</div>
+                <div class="font-semibold text-left">{{ txnStatus }}</div>
+              </div>
+              <div class="md:flex my-3 md:my-1">
+                <div class="md:w-56 text-left md:text-right mr-4">Sirius Recipient Address</div>
+                <div class="font-semibold text-left">{{ siriusAddress }}</div>
+              </div>
+              <div class="md:flex my-3 md:my-1">
+                <div class="md:w-56 text-left md:text-right mr-4">Amount Received</div>
```

Comment:
Maybe change to `Amount to be received`, the amount might be still pending or in partial to transfer over

Reply:
Okay, if the txn status is completed, will be displayed as Amount received, if not Amount to be received.

Reply:
complete as in Sirius site ? swap server will return complete when the ABT announced, it does not care about the success of the transfer

Reply:
Then will amend to just `Amount`. Is it ok?

Reply:
ok